.github/ISSUE_TEMPLATE/report-package-problems.yml

```diff
@@ -2,23 +2,36 @@
 name: Packaging problem report
 description: Report problems with a package. This includes build failures or other issues related to a package. This is *not* for bugs with installed software.
 labels: ["Package Problem", "Priority: Normal"]
-title: "packagename: problem"
+title: "packagename: problem with package"
 body:
   - type: checkboxes
     id: search
     attributes:
       label: Please confirm there isn't an existing open report for this package
       description: Before opening a new packaging problem report, please search [existing reports](https://github.com/getsolus/packages/labels/Package%20Problem%20Report) to ensure there is not an existing one.
-        options:
-          - label: I have searched through package problem reports
+      options:
+        - label: I have searched through package problem reports
             required: true
+  - type: checkboxes
+    id: ver_check
+    attributes:
+      label: Please make sure you are working with the latest version
+      description: If you have a problem building a package, or with the packaging of software, ensure you are running / building the latest version.
+      options:
+        - label: The package is the latest version
+          required: true
```

Comment:
I don't get the need of making sure if the package is the latest version. Some packages are kept to specific version deliberately and if they fail to build packagers should/can use this form to report it.

Reply:
Good point, I'm taking the checkbox out.